ui/store/actions.ts

```diff
@@ -1294,6 +1294,19 @@ export function revokeDynamicSnapPermissions(
   };
 }
 
+export function disconnectOriginFromSnap(
```

Comment:
It could be useful to keep a good practice and have a docstring here, despite this file is not following that too much.

Reply:
Done!